docs/source/notes/autograd.rst

```diff
@@ -863,6 +864,15 @@ the Tensor's grad_fn to be executed. For example, if the Tensor is passed
 as part of the ``inputs`` argument to :func:`torch.autograd.grad`,
 the Tensor's grad_fn may not be executed, but the hook register to that Tensor will always be executed.)
 
+Hooks registered to a Tensor via :meth:`torch.tensor.register_post_accumulate_grad_hook`
+are executed after the gradients have been accumulated for that Tensor, meaning the
+Tensor's grad field has been set. Whereas hooks registered via :meth:`torch.tensor.register_hook`
+are run as gradients are being computed, hooks registered via :meth:`torch.tensor.register_post_accumulate_grad_hook`
+wait to trigger on the accumulation of all gradient updates on the Tensor during the backward
+pass. Similar to node hooks, post accumulate grad hooks are only fired for leaf Tensors for which
```

Comment:
"node hooks" -> "hooks registered to Node"

also begin the sentence with "[This means that] similar to node hooks..." since this sentence is implied by the previous.